ddtrace/tracer.py

```diff
@@ -80,13 +90,16 @@ def configure(self, enabled=None, hostname=None, port=None, sampler=None,
             Otherwise they'll be dropped.
         :param str hostname: Hostname running the Trace Agent
         :param int port: Port of the Trace Agent
-        :param object sampler: A custom Sampler instance
+        :param object sampler: A custom Sampler instance, locally deciding to totally drop the trace or not.
+        :param object priority_sampler: A custom Sampler instance, taking the priority sampling decision.
         :param object context_provider: The ``ContextProvider`` that will be used to retrieve
             automatically the current call context. This is an advanced option that usually
             doesn't need to be changed from the default value
         :param object wrap_executor: callable that is used when a function is decorated with
             ``Tracer.wrap()``. This is an advanced option that usually doesn't need to be changed
             from the default value
+        :param priority_sampling: enable priority sampling, this is required for
```

Comment:
It makes the API a bit more complicated to have both `priority_sampler` and `priority_sampling`.

I guess the purpose was to simply allow to set `priority_sampling ` to turn it on?
Then I'd suggest to only keep this one. As no one will have to alter the `priority_sampler` ; let's only have `priority_sampling` as part of the API.